### DISCUSSION TITLE
Support : Error when exporting to NIF?

### DISCUSSION THREAD

**Discussion by robertmc8** [niftools/blender_niftools_addon#489] [Q&A]
@niftools/blender-niftools-addon-reviewer -

Before creating a new issue, ensure that
 - [ ] Read the user documentation.
 - [ ] Check that the issue hasn't already been reported.
 - [ ] Read through the list of `Helpful resources` links on the right-hand side.
 - [ ] Talk with the developers to ensure its a valid issue

The issue will get closed off immediately as invalid if
 - [ ] Looking for support where the forums, chat or documentation is more appropriate.
 - [ ] Goes off topic, loses focus or turns into a discussion.
 - [ ] Removing parts of the template the issue.

Fill out the template below to the best of your ability, including .blend files, nifs, logs; help us to help you.

# Issue Overview
**[Provide a general introduction to the issue and why it is relevant to this repository]**

### Blender Niftools Addon Version Info
**[Provide version information for the Blender Niftools Addon the issue occurs with, available in Blender in the Add-On section]**

### Blender Version Info
**[Provide information on which version of Blender the issue is seen]**

### Platform information
**[Provide a brief overview of what OS your running on]**

## Context
**[Provide more detailed information to the issue itself and why it is relevant]**

## Steps to Reproduce
**[Ordered list of the steps required for recreating the issue, including settings]**
 #. 

## Expected Result
**[Describe what you expected to have resulted from this process]**

## Actual Result
**[Describe what you currently experience from this process, and thereby explain the bug]**

## Possible Fix
**[Optional, suggest fixes, improvement or reasons for the bug might have occurred]**

## Screenshot
**[If relevant, include a screenshot]**

## Logs and Files
**[Provide logs file generated during the error as well as the blend and nif files are related to the issue]**

### Info Bar Output
**[Output from the Info View, available at top of Blender viewport, drag to expand]**
https://postimg.cc/ctW1xCm1

### Console Output
**[Set the logging level to 'Debug' and attach the output of the console. Enable via Window -> Toggle Console]**

### Blend File
**[Attach the blend file if the issue is reproducible]**

### Nif File
**[Attach input or output files, samples of what the expected output should be and reproducing the issue]**

## Similar Known Issues
**[Reference any known issues - https://github.com/niftools/blender_niftools_addon/issues]**

## Additional Information
**[Anything else you feel is relevant]**

Did I setup it right? 
https://postimg.cc/8f74CkP3
Bare with me this my first post!

**Comment by neomonkeus** [comment]
The information you have provided doesn't provide us any insight into your issue.
I have re-added the template so that you can provide the information we need in order to troubleshoot the problem, i.e the steps you took, the console logs output, etc. 

The screenshot of the report bar isn't useful, which is why in the template it mentions to ` drag to expand` so that we can see the actual report.
